integrations/aerospike/prometheus_metadata.yaml

```diff
@@ -13,6 +13,9 @@ platforms:
         prometheus_name: aerospike_namespace_memory_free_pct
         kind: GAUGE
         value_type: DOUBLE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/aerospike_namespace_device_available_pct/gauge
+        prometheus_name: aerospike_namespace_device_available_pct
```

Comment:
Reorder? Then LGTM

      - name: prometheus.googleapis.com/aerospike_namespace_device_available_pct/gauge
        prometheus_name: aerospike_namespace_device_available_pct
        value_type: DOUBLE

Reply:
Apologies, that was copy/paste error on my end, should be resolved.